notes/content/02_imgseg/oop.md

```diff
@@ -0,0 +1,17 @@
++++
+date = "2016-06-06T19:18:21+02:00"
+prev = "/02_imgseg/object-header"
+next = "/02_imgseg/object-formats/"
+title = "Object Oriented Pointer"
+weight = 20
++++
+
+When an object contains an oop it isn't even sure that this is a real pointer. Because the last digit of the oop shows if it refers a real object or if it is a so called small integer. A zero means that it is a real pointer, a one that it is a small integer.
```

Comment:
I think "the last digit" should be changed to something like "the last bit", "the LSB" or the like. At least for me, digit always implies decimal digit, unless it is explicitly stated otherwise.